demosplan/DemosPlanCoreBundle/Entity/Procedure/ProcedureSettings.php

```diff
@@ -902,8 +900,6 @@ public function getDesignatedPublicPhase()
 
     /**
      * @param string $designatedPublicPhase
-     *
-     * @return mixed
```

Comment:
if you remove the docblock type hint, then pls add it as an actual type hint. Which would be, in fact, better. :)

Reply:
I dont like what the style fixer does!
It wasn't me 4d62cca1cbf0553c05be7f209bee90ba2e429965